rpi-eeprom-update

```diff
@@ -576,6 +575,12 @@ findBootFS()
       TMP_BOOTFS_MNT="$(mktemp -d)"
       mount /dev/mmcblk0p1 "${TMP_BOOTFS_MNT}"
       BOOTFS="${TMP_BOOTFS_MNT}"
+   elif [ -z ${BOOTFS+set} ]; then
```

Comment:
"parameter expansion" - https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html

`${parameter+word}` becomes `word` if `$parameter` is set, but otherwise returns an empty string. It helps draw a distinction between an empty variable and an unset variable.

Supporting `BOOTFS=` makes no sense, so we could make it more readable by removing the use of a slightly obscure feature, if that's preferred.

Reply:
ok

By "Supporting BOOTFS= makes no sense" are you saying that it shouldn't be possible to set a hardcoded path via the defaults file (which seems useful in buildroot) or just that the previous BOOTFS default value made no sense?

Reply:
I mean that you wouldn't set BOOTFS to an empty string and expect things to work. So `-z "${BOOTFS}"`  would work just as well as `-z ${BOOTFS+set}` and would be easier to understand at a glance.

Reply:
ok 👍